python/paddle/fluid/layers/ops.py

```diff
@@ -61,7 +61,29 @@
 
 
 def uniform_random(shape, dtype=None, min=None, max=None, seed=None):
+    """
+    This operator initializes a variable with random values sampled from a
+    uniform distribution. The random result is in set [min, max].
+
+    Args:
+        shape (list): The shape of output variable.
+        dtype(np.dtype|core.VarDesc.VarType|str): The type of data, such as
+            float32, float64 etc. Default: float32.
```

Comment:
It may be not unified because the doc says the default value of `dtype` is `float32 `, but `dtype` is None in the parameter list.
http://paddlepaddle.org/documentation/docs/zh/1.2/api_cn/layers_cn.html#uniform-random

Reply:
@chengduoZH 

In following code,

```python
   if dtype is None:
        dtype = 'float32'
```

Here, I don't change the None to 'float32'.  Maybe should change.

Reply:
@chengduoZH I update the doc.